### DISCUSSION TITLE
Should the '{organization}' placeholder be replaced with something in the license?

### DISCUSSION THREAD

**Discussion by tpoliaw** [python-semver/python-semver#438] [General]
Hi, I'm vendoring a very stripped down version of python-semver for the comparison methods (just the `Version` class with all the bump methods missing) and while copying the license and copyright into the reduced file, I noticed the line

```
Neither the name of the {organization} nor the names of its contributors may be used to ...
```

Should that `{organization}` placeholder be replaced with something? Not sure what it'd be if this is a personal project. The [opensource.org version](https://opensource.org/license/BSD-3-clause) has `Neither the name of the copyright holder nor the names of its contributors...` instead.

**Comment by tomschr** [comment]
Thanks @tpoliaw! You are right! When I took over maintenance, I touched a lot of files, but the LICENSE wasn't on my list. :laughing: 

It's corrected in commit c47774d

**Comment by tpoliaw** [comment]
Thanks, I'll update the file header. And thanks for the library, it's saved me a lot of time.
